drivers/modem/ublox-sara-r4.c

```diff
@@ -41,16 +41,16 @@ enum mdm_control_pins {
 static struct modem_pin modem_pins[] = {
 	/* MDM_POWER */
 	MODEM_PIN(DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_CONTROLLER,
-		  DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_PIN, GPIO_DIR_OUT),
+		  DT_INST_0_UBLOX_SARA_R4_MDM_POWER_GPIOS_PIN, GPIO_OUTPUT),
 
 	/* MDM_RESET */
 	MODEM_PIN(DT_INST_0_UBLOX_SARA_R4_MDM_RESET_GPIOS_CONTROLLER,
-		  DT_INST_0_UBLOX_SARA_R4_MDM_RESET_GPIOS_PIN, GPIO_DIR_OUT),
+		  DT_INST_0_UBLOX_SARA_R4_MDM_RESET_GPIOS_PIN, GPIO_OUTPUT),
 
 #if defined(DT_UBLOX_SARA_R4_0_MDM_VINT_GPIOS_CONTROLLER)
 	/* MDM_VINT */
 	MODEM_PIN(DT_INST_0_UBLOX_SARA_R4_MDM_VINT_GPIOS_CONTROLLER,
-		  DT_INST_0_UBLOX_SARA_R4_MDM_VINT_GPIOS_PIN, GPIO_DIR_IN),
+		  DT_INST_0_UBLOX_SARA_R4_MDM_VINT_GPIOS_PIN, GPIO_OUTPUT),
```

Comment:
It looks like the direction has changed. I'm not sure if that's intentional or not (or if I'm reading this right).

Reply:
ops you are right! This is an oversight, will fix thanks!

Reply:
Fixed now, thanks again!